qa/TL0_jupyter/test_nofw.sh

```diff
@@ -15,9 +15,9 @@ test_body() {
     # test all jupyters except one related to a particular FW,
     # and one requiring a dedicated HW (multiGPU and OF)
     # optical flow requires TU102 architecture whilst this test can be run on any GPU
-    black_list_files="multigpu\|mxnet\|tensorflow\|pytorch\|paddle\|external_input.ipynb\|optical_flow\|python_operator\|#"
+    block_list_files="multigpu\|mxnet\|tensorflow\|pytorch\|paddle\|external_input.ipynb\|optical_flow\|python_operator\|#"
```

Comment:
That's screaming for typos from people trying to use it.
```suggestion
    exclude_files="multigpu\|mxnet\|tensorflow\|pytorch\|paddle\|external_input.ipynb\|optical_flow\|python_operator\|#"
```
However, that's not very inclusive. Pun intended.

Reply:
Done